.semaphore/semaphore.yml

```diff
@@ -58,8 +58,8 @@ blocks:
 
   - name: Live UI Gmail tests
     dependencies: []
-    run:
-      when: branch = 'master' OR branch =~ 'live-test'  OR branch =~ 'gmail-test'
+    # run:
+    #   when: branch = 'master' OR branch =~ 'live-test'  OR branch =~ 'gmail-test'
```

Comment:
Maybe we can now enable live test?

Reply:
Or you plan to now run live test every time?

Reply:
Oh, right, I forgot to rollback this change, thanks for noticing!